### PR TITLE
fix(deps): Include GitHub Dependabot scans/updates

### PR DESCRIPTION
**Pull request by superflyxxi**
Includes dependabot configuration so that yarn depenencies can be updated automatically (Pull Requests created). This will help with vulnerabilities.

### CODE REVIEW

**Comment by superflyxxi** [comment]
@kalinchernev do you think this can be merged? There are a lot of dependencies that have security vulnerabilities and this will help automatically resolve those.

**Review by daniloab — APPROVED** [review]
lets go with it

**Comment by daniloab** [comment]
waiting for this pr https://github.com/Surnet/swagger-jsdoc/pull/291 so we can move forward running the tests in this pull request.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+
+updates:
+  - package-ecosystem: "yarn"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```
Comment (daniloab) [inline .github/dependabot.yml:7]: what do you think about running daily the dependabot?

Comment (superflyxxi) [reply .github/dependabot.yml:7]: Sounds like a good idea. I'll make an update within the next hour.

**Review by daniloab — APPROVED** [review]

**Comment by superflyxxi** [comment]
@daniloab I think there may be an extra step needed to enable this. https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories

**Comment by superflyxxi** [comment]
@daniloab Are you an administrator of this Repo? Can you enable dependabot?

**Comment by daniloab** [comment]
> @daniloab Are you an administrator of this Repo? Can you enable dependabot?

cc @chdanielmueller

**Comment by chdanielmueller** [comment]
enabled `Dependabot alerts` and `Dependabot security updates`

**Pull request merged**